### PR TITLE
Todo List : CURD

### PR DESCRIPTION
**Pull request by ehmaddd**
It is the most recent update of the Todo List project. The salient features are that more modules have been added to the project to make it more dynamic and manageable. The previous version only had the front end with hard-coded entries in the list but now it is not the case, everything is dynamically loading and updating. Any task added, deleted, updated or deleted is immediately updated in localStorage and in the list as well. Delete buttons are provided to delete entries in a single. Moreover, in order to edit the description, the prompt option is used.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,22 +1,129 @@
...
+  const delBtn = document.querySelectorAll('.delete-btn');
+  for (let i = 0; i < delBtn.length; i += 1) {
+    delBtn[i].addEventListener('click', () => {
+      todo.taskArr.splice(i, 1);
+      todo.storeLocalStorage();
+      window.location.reload();
+    });
+  }
+};
+
```
Comment (DammyShittu) [inline src/index.js]: - [x] When a task is deleted the remaining task `index` values should be updated to newer values. If there are 5 tasks and I remove two tasks, regardless of the initial positions of the remaining tasks their new index values should be 1, 2, and 3 respectively.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @ehmaddd,

## Highlights
✔️ Index values start at 1

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (ehmaddd) [reply src/index.js]: Changes have been made according to the instruction. After deleting the item, index is re-adjusted successfully.

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**